lib/serializer.js

```diff
@@ -87,23 +87,22 @@ module.exports = class Serializer {
   }
 
   asString (str) {
-    const quotes = '"'
-    if (str instanceof Date) {
-      return quotes + str.toISOString() + quotes
-    } else if (str === null) {
-      return '""'
-    } else if (str instanceof RegExp) {
-      str = str.source
-    } else if (typeof str !== 'string') {
-      str = str.toString()
+    if (typeof str !== 'string') {
+      if (str === null) {
+        return '""'
+      } else if (str instanceof Date) {
+        return '"' + str.toISOString() + '"'
+      } else if (str instanceof RegExp) {
+        str = str.source
```

Comment:
```suggestion
      if (str === null) {
        return '""'
      }
      if (str instanceof Date) {
        return '"' + str.toISOString() + '"'
      } 
      if (str instanceof RegExp) {
        str = str.source
```

Just a nitpick to improve the legibility.